configure.ac

```diff
@@ -1437,6 +1437,21 @@ else
 fi
 
 ### Check for Qhull library.
+qhull_static_first=no
+AC_ARG_WITH([qhull-static],
```

Comment:
This is not what I meant. Please, try to read what I wrote before.

Instead of hardcoding two strings with potential names of the pkg-config file, please follow the existing pattern for the Magick++ libraries. I.e., use something like `--with-qhull_r="qhullstatic_r"`.

Reply:
Another question, if I have the flag with the new functionality. as I understand I will not need to use PKG_CHECK_EXISTS.
I will change the qhull_pc_name value, and pass it to the exist check of qhull. This is your intention?